bundles/org.openhab.binding.groheondus/README.md

```diff
@@ -54,13 +54,14 @@ and the `locationId`. Once the account bridge is configured, the appliances in y
 
 ##### senseguard
 
-| Channel                  | Type                     | Description                                           |
-|--------------------------|--------------------------|-------------------------------------------------------|
-| name                     | String                   | The name of the appliance                             |
-| pressure                 | Number:Pressure          | The pressure of your water supply                     |
-| temperature_guard        | Number:Temperature       | The ambient temperature of the appliance              |
-| valve_open               | Switch                   | Valve switch                                          |
-| waterconsumption         | Number                   | The amount of water used in a specific timeframe      |
+| Channel                         | Type                     | Description                                      |
+|---------------------------------|--------------------------|--------------------------------------------------|
+| name                            | String                   | The name of the appliance                        |
+| pressure                        | Number:Pressure          | The pressure of your water supply                |
+| temperature_guard               | Number:Temperature       | The ambient temperature of the appliance         |
+| valve_open                      | Switch                   | Valve switch                                     |
+| waterconsumption                | Number                   | The amount of water used in a specific timeframe |
+| waterconsumption_since_midnight | Number                   | The amount of water used since midnight          |
```

Comment:
Amount in which unit? Consider UoM by declaring `Number:Volume` and providing a unit (e.g. `Units.Litre`) with QuantityType when updating state. Since existing channel **waterconsumption** doesn't provide this either, please consider this only as a suggestion. Both channels should use same type.

Reply:
Agreed, but I could not find any information about this in the API. While it is selectable in the app (Metric vs Imperial) it could of course be added as a config parameter on the account

Reply:
Probably the API will provide the consumption in some concrete unit like L or m3? The app might convert it internally based on settings. openHAB will also provide automatic conversion if you support UoM.

Reply:
Using the metric option in the app, the result is in liters. Which I guess would be in gallons if I select imperial. But I could not find this field in the API classes. I assume this is why the original `waterconsumption` channel also is missing this info

Reply:
Given I didn't add the information into the API, the API is always returning the value in a "fixed" kind of metric. I would also assume that the app is then converting the values to the imperial system, if selected.

I can only guess what this metric here would be, I don't own such a device anymore and do not have access to one. However, my best guess would be liters to be honest.

Reply:
I would propose to figure this out, for example by checking if unit can be configured for the account, or if changing it in the app has any effect for openHAB. If API indeed always provides litre, then it would make sense to support UoM for clarity. Having a number without any documented unit seems not really usable?

Reply:
Appears that the service reports liters even if the app says Imperial. So added UoM.